test/unit/specs/components/monitor/PageBlock.spec.js

```diff
@@ -40,7 +40,9 @@ describe("PageBlock", () => {
     wrapper.update()
   })
 
-  it("has the expected html structure", () => {
+  it("has the expected html structure", async () => {
+    await wrapper.vm.$nextTick()
```

Comment:
Why do we need this here?

Reply:
$nextTick is added for the scrollbar plugin

Reply:
Could you maybe add comments describing this?

Reply:
In general the timing of component rendering, and the need for a lot of wrapper.updates() occurred after moving the components into an external library that had to be explicitly included into babel during the jest tests.